__mocks__/react.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Per this suggestion re async testing of useEffect:
+ * https://github.com/facebook/react/issues/14050#issuecomment-438173736
+ *
+ * If they ever fix that issue, we might be able to remove this mock.
+ */
+
+const React = require.requireActual('react');
```

Comment:
@marbemac you might want to change that either in Monaco repo.
I had to use `requireActual`, as I encountered an infinite recursion when mocking react using `jest.mock('react');`